tests/headers/complex_global.h

```diff
@@ -0,0 +1 @@
+double _Complex globalValue;
```

Comment:
With my fix it was possible to have saw_complex called for global, but not for member variables (or vice versa).
With your suggestion that won't be the case, but might be better to keep them separate.